ignite/contrib/metrics/mean_absolute_relative_error.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import division
+from ignite.exceptions import NotComputableError
+from ignite.metrics import Metric
+import torch
+
+
+class MeanAbsoluteRelativeError(Metric):
+    r"""
+    Calculate Mean Absolute Relative Error
+
+    :math:`\text{MARE} = \frac{1}{n}\sum _j^n\frac{\left|A_j-P_j\right|}{\left|A_j\right|}`
+
+    where, :math:`A_j` is the ground truth and :math:`P_j` is the predicted value.
+
+    More details can be found in the reference `Botchkarev 2018`_ .
```

Comment:
What is the difference between referencing as 
```rst
`text`__
__ https://link.to/your/target
```

vs 

```rst
`text`_
.. _ `text`:
    https://link.to/your/target
```
?
If we use the latter one for different metrics but with same text and url, would it create errors like in https://github.com/pytorch/ignite/pull/331#issuecomment-438177435